.php-cs-fixer.cache

```diff
@@ -1 +1 @@
iling_whitespace":true,"no_trailing_whitespace_in_comment":true,"single_blank_line_at_eof":true,"single_class_element_per_statement":{"elements":["property"]},"single_line_after_imports":true,"spaces_inside_parentheses":true,"statement_indentation":true,"switch_case_semicolon_to_colon":true,"switch_case_space":true,"encoding":true,"full_opening_tag":true,"array_syntax":{"syntax":"short"}},"hashes":{"app\/Blocks\/InsetText.php":"ca6db9fb90e6dc38036fd329b08b4feb","app\/Blocks\/WarningText.php":"4eedb41fdb8d9e32ed8f3a29954544e1","app\/Blocks\/Details.php":"229dc7f4122755f35c0eaa2ebc0870b2","app\/Blocks\/Accordion.php":"986baed18c174cc59523b49946f19c18","app\/Blocks\/NotificationBanner.php":"b0c0d71623d339eec4fc0392628202e5","app\/Blocks\/iBlock.php":"fb0cc3f4b55653e44e6f3268c62cea8e","app\/Blocks\/Button.php":"617d9e3be721d2ed486729ed5589bd34","app\/load.php":"a4f9b28a74fb6528b579d7b50938aced","app\/di.php":"fbc8c0cf5d51882f814d9e3a9edb4813","app\/BlockCategory.php":"96062756aec8f8c0ac62aeb742683535","app\/Options.php":"ed1ec0f3173d0f0c08eb92c2c35feb06","app\/BlockController.php":"3c99866fdc2a74b3e39958a7591eb5ee","index.php":"f601a517bea5cb1726fbaad021bc82b6","spec\/blocks\/details.spec.php":"7622ab8ff585d1fe3b7fc5f8eb650b8f","spec\/blocks\/notification_banner.spec.php":"b290162dd480e8296d6da4300eb02049","spec\/blocks\/accordion.spec.php":"479b91a049f5ebc17d137c91641b0eb6","spec\/blocks\/inset_text.spec.php":"ea301c7cbb898d25720b328e803a939f","spec\/blocks\/warning_text.spec.php":"0526586c5e5e9b4347ac8349e65ff4eb","spec\/blocks\/button.spec.php":"cb72508f9f8c2714fcd40f7720ca67b6","spec\/block_category.spec.php":"e86613f05b6946bbf4b9d4c80d12c303","spec\/options.spec.php":"9d1b130c5f025ca4066074158ee48e4b","spec\/block_controller.spec.php":"e7af5ba57db5557e73963e33ea123101","templates\/details.php":"9fcecf64b82d07f57abcb5bb0425a9d5","templates\/accordion.php":"6de8b940f86107693bf7a8e7c09aaf0b","templates\/notification_banner.php":"855648e14c87f7ff9405fd5f0dbecc6f","templates\/warning_text.php":"3cccd6d65c798e3a92fd745310f1c475","templates\/inset_text.php":"3e46980d3bdf2c2f842718b062dbbe7a","templates\/button.php":"737d9925258589c682fcd45af3beb57a","peridot.php":"6a778bf65a85942f2643a787215222a6","\/private\/var\/folders\/d8\/82xv262x6rnftct1fnngv65w0000gr\/T\/PHP CS Fixertemp_folder6\/templates\/accordion.php":"5624488a612dd6cf07ba2c5d4f78de10","\/private\/var\/folders\/d8\/82xv262x6rnftct1fnngv65w0000gr\/T\/PHP CS Fixertemp_folder861\/templates\/accordion.php":"5624488a612dd6cf07ba2c5d4f78de10","\/private\/var\/folders\/d8\/82xv262x6rnftct1fnngv65w0000gr\/T\/PHP CS Fixertemp_folder4\/templates\/accordion.php":"6de8b940f86107693bf7a8e7c09aaf0b","\/private\/var\/folders\/d8\/82xv262x6rnftct1fnngv65w0000gr\/T\/PHP CS Fixertemp_folder1659\/templates\/accordion.php":"6de8b940f86107693bf7a8e7c09aaf0b","\/private\/var\/folders\/d8\/82xv262x6rnftct1fnngv65w0000gr\/T\/PHP CS Fixertemp_folder1321\/templates\/accordion.php":"6de8b940f86107693bf7a8e7c09aaf0b"}}
```

Comment:
Can we please add a new commit to delete this file and put it in `.gitignore`?

Reply:
Done